.travis.yml

```diff
@@ -2,8 +2,26 @@ language: java
 before_script:
 - unset _JAVA_OPTIONS
 - cd finish
+- chmod +x ../scripts/travisTest.sh
 script:
-- mvn -q clean install
+- ../scripts/travisTest.sh
+after_script:
+- logsPath=$(find . -name "console.log" | sed "s/console.log//g");
+- build=$(grep "Open Liberty" $logsPath/console.log | cut -d" " -f5 | cut -d")" -f1 );
+- release=$( echo "$build" | cut -d'/' -f1); 
+- number=$(echo "$build" | cut -d'/' -f2); 
+- ol_jv=$(grep -i "version" $logsPath/console.log);
+- jv=$(printf '%s\n' "${ol_jv//$' on '/$'\n'}" | sed '2q;d'); 
+- echo -e "\n"; \
+  echo -e "\033[1;34m Open Liberty release:\033[0m\033[1;36m $release \033[0m \n"; \
+  echo -e "\033[1;34m Open Liberty build number:\033[0m\033[1;36m $number \033[0m \n"; \
+  echo -e "\033[1;34m Java version:\033[0m\033[1;36m $jv \033[0m \n";
+- cd $logsPath;
+- repo_name=$(echo "$TRAVIS_REPO_SLUG" | sed -e "s/\//-/g"); 
+- if [ "$TRAVIS_TEST_RESULT" -eq 0 ]; then result="passed"; else result="failed"; fi; 
+- serverlogsarchive="$repo_name-$TRAVIS_BUILD_NUMBER-$result.zip";
+- zip -r "$serverlogsarchive" .;
+- curl -H "$JFROG_TOKEN" -T "$serverlogsarchive" "https://na.artifactory.swg-devops.com/artifactory/hyc-openliberty-guides-files-generic-local/";
```

Comment:
Travis tests are failing on this PR. I looked into it and it fails on generating the logs via this `after_script` section. Verify that the script generates a `console.log` file for logging purposes. If not, then delete this whole section so that the Travis tests don't fail.